FCCee/ALLEGRO/compact/ALLEGRO_o1_v03/HCalEndcaps_ThreeParts_TileCal.xml

```diff
@@ -44,20 +44,23 @@
   <readouts>
     <!-- Default readout of DetailedWedge geometry -->
     <readout name="HCalEndcapReadout">
-      <segmentation type="FCCSWGridPhiEta_k4geo" grid_size_eta="0.025" phi_bins="EndcapHCal_n_phi_modules" offset_eta="-3.5" offset_phi="-pi+(pi/EndcapHCal_n_phi_modules)"/>
-      <id>system:4,type:2,layer:5,row:9,eta:11,phi:10</id>
+      <segmentation type="FCCSWGridPhiTheta_k4geo" grid_size_theta="0.022180" phi_bins="EndcapHCal_n_phi_modules" offset_theta="0.06" offset_phi="-pi+(pi/EndcapHCal_n_phi_modules)"/>
+      <id>system:4,type:2,layer:5,row:9,theta:11,phi:10</id>
     </readout>
-   <readout name="HCalEndcapReadout_phieta">
-     <segmentation type="FCCSWGridPhiEta_k4geo" grid_size_eta="0.025" offset_eta="-3.5" phi_bins="256" offset_phi="-pi+pi/256."/>
-     <id>system:4,type:2,layer:4,eta:11,phi:10</id>
+   <readout name="HCalEndcapReadout_phitheta">
+     <segmentation type="FCCSWGridPhiTheta_k4geo" grid_size_theta="0.022180" phi_bins="EndcapHCal_n_phi_modules" offset_theta="0.06" offset_phi="-pi+(pi/EndcapHCal_n_phi_modules)"/>
+     <id>system:4,type:2,layer:5,theta:11,phi:10</id>
     </readout>
    <readout name="HCalEndcapReadout_phi">
-     <!-- the eta segmentation here is a dummy - the whole hcal is contained in one bin. the z position is stored in the "row" field. -->
-     <segmentation type="FCCSWGridPhiEta_k4geo" grid_size_eta="4." phi_bins="EndcapHCal_n_phi_modules" offset_eta="-3.5" offset_phi="-pi+(pi/EndcapHCal_n_phi_modules)"/>
-     <id>system:4,type:2,layer:5,row:9,eta:10,phi:10</id>
+     <segmentation type="FCCSWGridPhiTheta_k4geo" grid_size_theta="3." phi_bins="EndcapHCal_n_phi_modules" offset_theta="0.06" offset_phi="-pi+(pi/EndcapHCal_n_phi_modules)"/>
+     <id>system:4,type:2,layer:5,row:9,theta:0,phi:10</id>
```

Comment:
Are those two readouts still needed?

Reply:
They are not used for the moment, but I'd prefer to keep them to be on the safe side in case they'd be useful in the future. For the HCal barrel, we also keep three readouts.

Reply:
I think the _phi readout is not needed, so we might get rid of it if cleanup is needed. phi-theta might be needed for clustering, so I'd rather keep if for the time being till we sort out exactly which cells we use for the clustering (do we use cells based on the physical row-phi arrangement or based on a projective theta-phi grid?)

Reply:
_phi readout removed from both barrel and endcap xml